pkg/securitypolicy/securitypolicyenforcer_rego.go

```diff
@@ -531,36 +560,28 @@ func (policy *regoEnforcer) updateMetadata(results map[string]interface{}) error
 	return nil
 }
 
-func (policy *regoEnforcer) enforce(enforcementPoint string, input map[string]interface{}) error {
+func (policy *regoEnforcer) enforce(enforcementPoint string, input map[string]interface{}) (map[string]interface{}, error) {
```

Comment:
can you document what is being returned here? it should be the processed policy that matches the given the inputs, correct?

Reply:
Given the proliferation of `map[string]interface{}` as a result of working with Rego data stores and results (which are stored in this way) I've done a small refactor in which every raw `map[string]interface{}` has, where possible, been replaced with a type that gives context on what it is. For example, this is now `policyResults`.